lib/src/sql/function.rs

```diff
@@ -271,8 +271,10 @@ fn cast(i: &str) -> IResult<&str, Function> {
 fn function_names(i: &str) -> IResult<&str, &str> {
 	recognize(alt((
 		preceded(tag("array::"), function_array),
+		preceded(tag("bytes::"), function_bytes),
 		preceded(tag("crypto::"), function_crypto),
 		preceded(tag("duration::"), function_duration),
+		preceded(tag("encoding::"), function_encoding),
 		preceded(tag("geo::"), function_geo),
```

Comment:
`encoding` and `geometry` are both 8 letter words. Do you think it would be better to abbreviate `encoding` to `enc` or leave it un-abbreviated like `duration`?